script/tool/test/common_test.dart

```diff
@@ -298,10 +268,10 @@ packages/plugin1/plugin1/plugin1.dart
 packages/plugin1/plugin1_platform_interface/plugin1_platform_interface.dart
 packages/plugin1/plugin1_web/plugin1_web.dart
 ''';
-      final Directory plugin1 = createFakePlugin('plugin1',
-          parentDirectoryName: 'plugin1', packagesDirectory: packagesDir);
-      createFakePlugin('plugin2', packagesDirectory: packagesDir);
-      createFakePlugin('plugin3', packagesDirectory: packagesDir);
+      final Directory plugin1 = createFakePlugin('plugin1', packagesDir,
```

Comment:
Nit: It's a bit of a step backward removing the label to the argument.  We could  have made it required instead of making it an argument without a label.

Reply:
That's what I did initially, and essentially every call was "packagesDirectory: packagesDir", which just seemed noisy, so I made it positional.